code/datums/ai_laws.dm

```diff
@@ -109,7 +109,7 @@
 	name = "Thermodynamic"
 	id = "thermodynamic"
 	inherent = list("The entropy of the station must remain as constant as possible.", \
-					"The entropy of the station always endeavours to increase.", \
+					"The entropy of the station always endeavors to increase.", \

```

Comment:
Remember to replace the ess with a zed to trigger more.

Reply:
Didn't realize this one actually was GB spelling, would'cha look at that.

Reply:
Whatever the maintainers say goes, I don't know if we still have the no touchy rule from when Remie was a maintainer. I'm just reeing.